### ISSUE TITLE
How to test from localhost website?

### ISSUE THREAD

**Issue by bennyn** [HyperCubeProject/binance-api-node#152]
I bundled `'binance-api-node'` and tried to use it within a website but whenever I do a request, I am receiving the following error:

> Access to fetch at 'https://api.binance.com/api/v1/time' from origin 'http://localhost:8080' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled

**Comment by bennyn** [comment]
I tried to run Chrome with `--disable-web-security` to bypass the need of a Proxy server but it didn't work. 

Would you accept a PR that informs about the CORS issue in the `README.md`? In another issue you [pointed out](https://github.com/HyperCubeProject/binance-api-node/issues/80#issuecomment-378687538) that the library must be Browser-compatible but with the CORS issue this seem to become less of importance?

**Comment by SrBrahma** [comment]
I answered my solution here: https://github.com/Ashlar/binance-api-node/issues/37#issuecomment-828132105
